data/xml/Q20.xml

```diff
@@ -0,0 +1,171 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="Q20">
+  <volume id="1">
+    <meta>
+      <booktitle>Transactions of the Association for Computational Linguistics, Volume 8</booktitle>
+      <year>2020</year>
+    </meta>
+    <frontmatter />
+    <paper id="1">
+      <title>Phonotactic Complexity and Its Trade-offs</title>
+      <author><first>Tiago</first><last>Pimentel</last></author>
+      <author><first>Brian</first><last>Roark</last></author>
+      <author><first>Ryan</first><last>Cotterell</last></author>
+      <doi>10.1162/tacl_a_00296</doi>
+      <abstract>We present methods for calculating a measure of phonotactic complexity—bits per phoneme— that permits a straightforward cross-linguistic comparison. When given a word, represented as a sequence of phonemic segments such as symbols in the international phonetic alphabet, and a statistical model trained on a sample of word types from the language, we can approximately measure bits per phoneme using the negative log-probability of that word under the model. This simple measure allows us to compare the entropy across languages, giving insight into how complex a language’s phonotactics is. Using a collection of 1016 basic concept words across 106 languages, we demonstrate a very strong negative correlation of − 0.74 between bits per phoneme and the average length of words.</abstract>
+      <pages>1–18</pages>
+      <url hash="c153b336">Q20-1001</url>
+    </paper>
+    <paper id="2">
+      <title>AMR-To-Text Generation with Graph Transformer</title>
```

Comment:
Evidently the truecaser hasn't been run—it would flag AMR

Reply:
@aryamccarthy, the relevant code in `ingest.py` is here:

```python
                for oldnode in paper_node:
                    normalize(oldnode, informat="latex")

                for name_node in chain(
                    paper_node.findall("./author"), paper_node.findall("./editor")
                ):
                    disambiguate_name(name_node, paper_id_full)
```

I run this on every node I create for a new paper, which adds title casing protect and (with manual intervention) disambiguates the authors.